port/action-permissions/actionPermissionToPortBody.go

```diff
@@ -6,6 +6,22 @@ import (
 	"github.com/port-labs/terraform-provider-port-labs/internal/flex"
 )
 
+func policyToPortBody(policy *string) (*map[string]any, error) {
```

Comment:
move this to be with the rest xToPortBody functions and rename it to jsonStringToPortBody

i mean its not specific to policy - it can work for other json properties

Reply:
will update in a followup PR